build/common.project.props

```diff
@@ -113,10 +112,6 @@
     <VsixPublishDestination>$(ArtifactRoot)$(VsixOutputDirName)\</VsixPublishDestination>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(BuildRTM)' == 'true' ">
```

Comment:
in every case, rtm and non-trm builds will have the same output directory right?

Reply:
Yes, that's the goal. On CI we run the RTM and non-RTM builds on different jobs, so there's never a scenario where both builds need to be available side-by-side.  After talking to someone in dnceng, it appears that other teams manually queue a build of a specific commit, with a variable set to their equivalent of `BuildRTM`, so they don't even do dual RTM/nonRTM builds as we do.  I'm considering changing NuGet.Client in the same way, but not in this PR.

Reply:
Resolving this conversation.